frontend/public/module/k8s/get-resources.ts

```diff
@@ -82,16 +82,12 @@ export type DiscoveryResources = {
   safeResources: string[];
 };
 
-export const kindToLabel = (kind: string): string => {
-  return _.startCase(kind).replace(/\bO Auth\b/, 'OAuth');
-};
-
 export const pluralizeKind = (kind: string): string => {
-  const label = kindToLabel(kind);
-  const pluralized = plural(label);
+  // Use startCase to captialize but remove spaces so as not to humanize
```

Comment:
It's not really to capitalize, more to break it into words so `plural` can pluralize the last word.

We should make sure this is working as intended. For instance, does `FooIdentity` become `FooIdentities`?

Reply:
Thanks, Sam.

> It's not really to capitalize, more to break it into words so `plural` can pluralize the last word.

Comment updated.  That better?

> We should make sure this is working as intended. For instance, does `FooIdentity` become `FooIdentities`?

[`ImageContentSourcePolicy`](https://github.com/openshift/console/pull/5038/files#diff-d6d1b521b27a0f5e246e03499fb9a289R16) is a good existing test case for multiple words.  But I also manually tested `FooIdentity` and a few others.

<img width="1174" alt="Screen Shot 2020-04-14 at 5 31 07 PM" src="https://user-images.githubusercontent.com/895728/79276509-2c23b600-7e76-11ea-950a-6eefa36acf45.png">